resources/assets/js/services/twitter/Tweet.js

```diff
@@ -2,10 +2,14 @@ import { get } from 'lodash';
 import moment from 'moment';
 import twemoji from 'twemoji';
 
-export default class {
+class Tweet {
```

Comment:
To be able to create a new instance of the Tweet class for a quote (in the class' constructor):
```js
this.quotedTweet = new Tweet(this.tweetProperties.quoted_status);
```
I couldn't find another way to create a new instance of an unnamed class